dotCMS/src/main/java/com/dotmarketing/business/ajax/PermissionAjax.java

```diff
@@ -419,30 +429,31 @@ private Permissionable retrievePermissionable (String assetId, Long language, Us
 		return perm;
 	}
 
-	  public void permissionIndividually(String assetId, Long languageId) throws Exception {
-			HibernateUtil.startTransaction();
-	    	try {
-	    		UserWebAPI userWebAPI = WebAPILocator.getUserWebAPI();
-	    		WebContext ctx = WebContextFactory.get();
-	    		HttpServletRequest request = ctx.getHttpServletRequest();
-
-	    		//Retrieving the current user
-	    		User user = userWebAPI.getLoggedInUser(request);
-	    		boolean respectFrontendRoles = !userWebAPI.isLoggedToBackend(request);
-
-	    		PermissionAPI permissionAPI = APILocator.getPermissionAPI();
-	    		Permissionable asset = retrievePermissionable(assetId, languageId, user, respectFrontendRoles);
-	    		Permissionable parentPermissionable = APILocator.getPermissionAPI().findParentPermissionable(asset);
-
-	    		if(parentPermissionable!=null){
-	    			permissionAPI.permissionIndividually(parentPermissionable, asset, user);
-	    		}
-	    		HibernateUtil.closeAndCommitTransaction();
-	    	} catch (Exception e) {
-	    		HibernateUtil.rollbackTransaction();
-	    		throw e;
-	    	}
-	    }
+	public void permissionIndividually(String assetId, Long languageId) throws Exception {
+		HibernateUtil.startTransaction();
+	   	try {
+	   		UserWebAPI userWebAPI = WebAPILocator.getUserWebAPI();
+	   		WebContext ctx = WebContextFactory.get();
+	   		HttpServletRequest request = ctx.getHttpServletRequest();
+
+	   		//Retrieving the current user
+	   		User user = userWebAPI.getLoggedInUser(request);
+	   		boolean respectFrontendRoles = !userWebAPI.isLoggedToBackend(request);
+
+	   		PermissionAPI permissionAPI = APILocator.getPermissionAPI();
+	   		Permissionable asset = retrievePermissionable(assetId, languageId, user, respectFrontendRoles);
+	   		Permissionable parentPermissionable = APILocator.getPermissionAPI().findParentPermissionable(asset);
+
+	   		if(parentPermissionable!=null){
+	   			permissionAPI.permissionIndividually(parentPermissionable, asset, user);
+	   		}
+	   		HibernateUtil.closeAndCommitTransaction();
```

Comment:
this could be in a finally or wrap in a WrapInTransaction

Reply:
This code was not changed at all, I just fixed the indentation. However, I can add the annotation as well.